modules/apps/site-initializer/site-initializer-masterclass/site-initializer-masterclass-test/src/testFunctional/tests/Masterclass.testcase

```diff
@@ -1110,6 +1110,62 @@ definition {
 		}
 	}
 
+	@description = "This is a test for LPS-148457. Masterclass create account and forgot password are not displayed in sign in page"
+	@priority = "3"
+	test MasterclassSignInPageDoesNotDisplayCreateAndForgotPage {
+		property portal.acceptance = "true";
```

Comment:
do we want to add acceptance to a propriety 3 test?

Reply:
Nop. copy&paste error.